packages/orchestration/src/examples/stakeAtom.contract.js

```diff
@@ -99,7 +101,8 @@ export const start = async (zcf, privateArgs, baggage) => {
     {
       async makeAccount() {
         trace('makeAccount');
-        return makeAccount().then(({ account }) => account);
+        const { account } = await makeAccount();
```

Comment:
if `account` is a property of the return value rather than the return value itself, the function should be called `makeAccountKit`, not `makeAccount`.